test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -643,9 +648,13 @@ private static JComponent createInstructionUIPanel(String instructions,
         JTextComponent text = instructions.startsWith("<html>")
                               ? configureHTML(instructions, rows, columns)
                               : configurePlainText(instructions, rows, columns);
+        if (hyperlinkListener != null && text instanceof JEditorPane ep) {
+            ep.addHyperlinkListener(hyperlinkListener);
+        }
         text.setEditable(false);
         text.setBorder(createTextBorder());
         text.setCaretPosition(0);
+        text.getCaret().setVisible(false);
```

Comment:
Hiding the caret could be undesirable: if a tester wants to select text using keyboard, there's no way to see where the caret is.

The caret used to be hidden for read-only text components, now it's displayed by default but it doesn't blink, see [JDK-8299077](https://bugs.openjdk.org/browse/JDK-8299077).